Root/XaodAnalysis.cxx

```diff
@@ -869,8 +924,59 @@ bool XaodAnalysis::eleIsOfType(const xAOD::Electron &in, eleID id)
 void XaodAnalysis::fillEventTriggers()
 {
     if(m_dbg>=5) cout << "fillEventTriggers" << endl;
-
-    m_evtTrigFlags = 0;
+   
+    m_evtTrigFlags = 0; 
+
+    if(m_trigTool->isPassed("EF_e7_medium1"))                   m_evtTrigFlags |= triggerbits::TRIG_e7_medium1;
```

Comment:
Can we actually drop the `m_evtTrigFlags` completely, and only have the
implementation with `m_evtTriggerBits`? The `|=` is stil less clear than `SetBitNumber`
and having two "parallel" implementations (one with `long long` and one with `TBits`)
is confusing.